test/mail_dhparams_manual_not_one_dir.bats

```diff
@@ -13,29 +13,29 @@ load 'test_helper/common'
 
 
 function setup() {
-    run_setup_file_if_necessary
+  run_setup_file_if_necessary
 }
 
 function teardown() {
-    run_teardown_file_if_necessary
+  run_teardown_file_if_necessary
 }
 
 function setup_file() {
-    PRIVATE_CONFIG=$(duplicate_config_for_container .)
-    # copy the custom DHE params in local config
-    cp "`pwd`/test/test-files/ssl/custom-dhe-params.pem" "${PRIVATE_CONFIG}/dhparams.pem"
+  local PRIVATE_CONFIG=$(duplicate_config_for_container .)
```

Comment:
Beware: `local var =$()` may hide failures in the `$()` part. Shellcheck should warn about that. I just learned  that shellcheck 0.7+ is able to check .bats files.

Reply:
I just committed in a hurry. I know about this and will fix it after lunch. I guess the fix is just:

``` BASH
local VAR
VAR="$(...)"
```

And, yeah, I saw some lints for `.bats` files. I can include them in the lint targets if you'd like.

Reply:
It would be nice to have those lints but for now they must not fail the build or we will be swamped. These files have never been checked before, so they have a huge amount of messages.

I already caught and fixed one error through this in tests.bats

Reply:
This PR has become bigger than I originally anticipated. I will enable lints for these files later:)